packages/harmony/src/components/avatar/Avatar.tsx

```diff
@@ -30,14 +30,18 @@ export const Avatar = (props: AvatarProps) => {
   const rootCss: CSSObject = {
     height: sizeMap[size],
     width: sizeMap[size],
+    overflow: 'hidden',
     borderRadius: 'calc(infinity * 1px)',
     border: `${strokeWidthMap[strokeWidth]} solid ${color.border.default}`,
     boxShadow:
       variant === 'strong'
         ? '0px 0.5px 1.5px 0px rgba(0, 0, 0, 0.03), 0px 1.5px 5px 0px rgba(0, 0, 0, 0.08), 0px 6px 15px 0px rgba(0, 0, 0, 0.10)'
         : 'none',
     backgroundImage: `url(${src})`,
-    backgroundColor: 'unset',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: src !== undefined ? 'unset' : '#C2C0CC',
```

Comment:
would a truthy check be fine here?
```suggestion
    backgroundColor: src ? '#C2C0CC' : 'unset',
```

Reply:
I dig it